.github/workflows/gh-actions-pr.yml

```diff
@@ -7,8 +7,12 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
 
+    defaults:
+      run:
+        shell: bash
+
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
we probably want fast-fail as it prevents other builders from running if earlier ones fail.

Reply:
Fixed. That was supposed to be temporary. Good catch